src/github.com/getlantern/flashlight/status_pages/generic_error.html

```diff
@@ -0,0 +1,115 @@
+
+      <h1>Address not found</h1>
```

Comment:
Again here -- "Error accessing page" perhaps? We don't know it was because the address wasn't found at this point.